tasks/main.yml

```diff
@@ -81,6 +81,12 @@
   sudo: yes
   template: src=kube-controller-manager.yaml dest={{kube_manifests_dir}}/kube-controller-manager.yaml
 
+- name: ensures kube scheduler policy config dir exists
```

Comment:
755 should be fine. As the "core" user might want to read. Maybe 750 is fine too. Approving.